cmd/kogito/command/message/kogitoBuild.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
```

Comment:
Please rename the file to `kogitobuild.go`

Reply:
done